### DISCUSSION TITLE
is service station mothballed?

### DISCUSSION THREAD

**Discussion by kleerkoat** [knurling/ServiceStation#68] [General]
i wish i checked GitHub before i purchased as it seems like development has stopped. i hope i'm not right.

**Comment by gingerbeardman** [comment]
Is it not working for you? 

If it is working then rest assured it's a good purchase.

**Comment by gilby101** [comment]
Certainly looks like development has stopped.  No issues or suggestions have been addressed for a long time.

Nevertheless, it does work as is and has survived the upgrade of macOS to Monterey without an update.  Let's hope it survives the macOS 13 upgrade.

**Comment by 2Data6** [comment]
It is February 16th 2024 as I write this and the Mac M3 was released in October 2024.  I am wondering if ServiceStation is still attended to and in development.  Even though some have said it works in 2022 I am hesitant to purchase it if no further attention is being given to it in the future.  Thanks.

**Reply by gilby101** [reply]
I have changed to Context Menu. Similar, but different. Last updated 3 months ago. Documentation minimal.
